src/web/services/navigation.service.ts

```diff
@@ -53,4 +74,11 @@ export class NavigationService {
     }
     window.open(url.toString());
   }
+
+  /**
+   * Opens a new browser window with URL Param encoding
+   */
+  openNewWindowWithParamEncoding(urlWithoutParams: string, params: {[key: string]: string}): void {
```

Comment:
`WithParamEncoding` should not be part of the name, as it kind of suggests that "param encoding" is some kind of isolated case. The fact is, param encoding should be expected especially when any kind of user input is involved.

Would suggest to combine this with the previous method and set `{}` to be the default value for `params`.

Reply:
Should the same apply to `navigateWithSuccessMessage`?

Reply:
If possible.